packages/medusa/src/migrations/1672906846559-category.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class category1672906846559 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE "category"
+        (
+          "id" character varying NOT NULL,
+          "name" text NOT NULL,
+          "handle" text NOT NULL,
+          "parent_category_id" character varying,
+          "mpath" text,
+          "is_active" boolean DEFAULT false,
+          "is_internal" boolean DEFAULT false,
+          "deleted_at" TIMESTAMP WITH TIME ZONE,
+          "created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+          "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+          CONSTRAINT "PK_qgguwbn1cwstxk53efl0px9oqwt" PRIMARY KEY ("id")
+        )
+    `)
+
+    await queryRunner.query(
+      `CREATE UNIQUE INDEX "IDX_b0dhljrfjktg1e98cnub69n502" ON "category" ("handle") WHERE deleted_at IS NULL`
```

Comment:
suggestion(non blocking): We can give a more understandable name to the index wdyt? `IDX_unique_category_handle_not_deleted`

Reply:
Would really love that. I was following existing methods. shall we?

Reply:
Personally when i write a migration i put some Hunan readable name composed as suggested to avoid conflicts. Otherwise when you get an error it is hard to understand where it comes from since the ids are random. Up to you but i would do that